src/zimcheck.cpp

```diff
@@ -92,27 +92,29 @@ static std::unordered_map<TestType, std::pair<LogTag, std::string>> errormapping
 class ErrorLogger {
   private:
     std::unordered_map<TestType, std::vector<std::string>> errors;
-    std::unordered_map<TestType, bool> testStatus;
+    std::unordered_map<TestType, std::pair<bool, bool>> testStatus;
```

Comment:
This change is a bit complex to understand (Especially without any comments or commit message).

More than a commit message, it would be better to create a structure to store a test status and then have named member instead of first and second.